presto-native-execution/BUILD_ON_UBUNTU.md

```diff
@@ -0,0 +1,201 @@
+# Develop and Test Prestissimo on Ubuntu
+
+On Ubuntu 20.04 or 22.04 (other versions may work but not tested), clone the whole Presto repository and run the setup script for Prestissimo (i.e., the C++ worker):
+```bash
+git clone --recurse-submodules https://github.com/prestodb/presto.git
+cd presto/presto-native-execution
+sudo ./scripts/setup-ubuntu.sh
+```
+
+Ensure that maven (3.3.9+) and JDK8 (8u151+) are installed and in-use, they are required for building the Java codebase (coordinator and client, etc.).
+
+## Build Java Codebase
+
+In the root directory of the presto repository, build the Java codebase:
+```bash
+mvn clean install -DskipTests -T1C -pl -presto-docs
+```
+
+## Setup Development Environment in IDEs
```

Comment:
@bianhq There is a lot of overlap with the instructions here https://github.com/prestodb/presto/tree/master/presto-native-execution#development-env
Can you clarify what is specific to Ubuntu?

Reply:
Yes, the IDE setup is mainly from README.md. I revised it a bit to make it more clear.
If we decide to maintain a single md file, I can merge the content here into README.md.